netmiko/ssh_autodetect.py

```diff
@@ -166,6 +166,12 @@
         "priority": 99,
         "dispatch": "_autodetect_std",
     },
+    "h3c": {
```

Comment:
There is no `h3c` device_type in Netmiko. 

This probably should return `hp_compware` or we should just put in another key to the `ssh_dispatcher.py` mapping `h3c_compware` to the `HPComwareSSH` class.

We should probably do the latter.

Do you want to implement that fix? If not, I will update this PR and implement it (just let me know).